Gallery/Public/Item Implementations/ViewControllerItem.swift

```diff
@@ -25,33 +25,52 @@
 import UIKit
 
 public struct ViewControllerItem: Item {
```

Comment:
I can do that, but that I will effectively duplicate ViewControllerItem and add this new fields because we need everything + snapshot. Is there any value in extracting this separate item?

Reply:
I don't have a strong argument about that, other than to keep this implementation simpler. In case you'd want to add some additional properties to snapshotted view controller for example, this one wouldn't be affected